kale/pipeline/mpca_trainer.py

```diff
@@ -0,0 +1,165 @@
+# =============================================================================
+# Author: Shuo Zhou, szhou20@sheffield.ac.uk
+#         Haiping Lu, h.lu@sheffield.ac.uk or hplu@ieee.org
+# =============================================================================
+
+"""Implementation of MPCA->Feature Selection->Linear SVM/LogisticRegression Pipeline
```

Comment:
Add references of papers using this pipeline: 
For Cardiac MRI: https://doi.org/10.1093/ehjci/jeaa001
For brain fMRI: https://doi.org/10.1007/978-3-319-24553-9_75
For gait videos (though KNN rather than SVM/LR was used): https://doi.org/10.1109/TNN.2007.901277